torch/_inductor/config.py

```diff
@@ -814,6 +815,52 @@ class cuda:
     cutlass_op_denylist_regex: Optional[str] = "pingpong"
 
 
+class rocm:
+    # Device arch list, e.g. ["gfx908", "gfx942"]. If empty, the `native` arch is used
+    arch: List[str] = []
+
+    # Hip (and ROCm) version, if None, will use torch.hip.version
+    hip_version: Optional[str] = None
```

Comment:
when would you want this to be different than `torch.hip.version`?

Reply:
It's unused for now so let me remove it to avoid confusion. Hypothetically we could have different rocm distributions for pytorch compilation and the cpp sources compilation with this backend.